operations/mimir-mixin/dashboards/writes.libsonnet

```
@@ -162,10 +162,39 @@ local filename = 'mimir-writes.json';
           'histogram_quantile(0.99, sum by(le, %s) (rate(cortex_request_duration_seconds_bucket{%s, route=~"/distributor.Distributor/Push|/httpgrpc.*|%s"}[$__rate_interval])))' % [$._config.per_instance_label, $.jobMatcher($._config.job_names.distributor), $.queries.write_http_routes_regex], ''
         )
       )
+      .addPanelIf(
+        $._config.show_ingest_storage_panels,
+        $.timeseriesPanel('ingest storage: WriteSync latency') +
```

[nit] A couple of things:

1. We're used to begin the title with an uppercase letter
2. I think the most important info here is "writesync latency" rather then "ingest storage", so I would move "ingest storage" to the end
3. "WriteSync" is an internal thing. We can probably generalise it as "Sync write to Kafka"

```suggestion
        $.timeseriesPanel('Sync write to Kafka latency (ingest storage)') +
```